lib/pbench/server/api/resources/query_apis/query_controller_list.py

```diff
@@ -107,9 +72,14 @@ def post(self):
         name, the number of runs using that controller name, and the start
         timestamp of the latest run both in binary and string form.
 
-        TODO: We probably need to return all *unowned* controllers when called
-        without a session token or user parameter. We need to define precisely
-        how this should work.
+        TODO: Do we automatically return all "public" controllers, or should
+        this be a separate parameter? E.g., when "I" log in, do I see all my
+        own datasets *plus* all published/public controllers owned by others,
+        or should that be a separate option (e.g., a checkbox on the view?)
+
+        TODO: Similarly, a query from an unlogged-in session in the dashboard
+        would presumably show datasets with "user": "public"; but does it show
+        all datasets with "access": "public" as well?
```

Comment:
Wouldn't all the dataset with "public" user have "access" public as well? how are they 2 different things?